KHANAAS-GUIDE.md

```diff
@@ -79,7 +79,41 @@ vagrant ssh -c './manage.py createsuperuser'
  - View the page at http://localhost:8000/hello
 
 #### Create simple API views
- - create urls for khan/$name and spock/$name, view/template that display $name on the page
+ - Create a html template, view and url that display a phrase on the page
+   - Create a html template by creating `api/templates/kirk.html` and add the following code:
```

Comment:
I believe this should be `api/templates/api/kirk.html`, i'll fix that in my current PR after I rebase.